src/org/jitsi/impl/neomedia/rtcp/RTCPReceiverFeedbackTermination.java

```diff
@@ -289,13 +289,17 @@ private long getSenderSSRC()
 
                 if (logger.isTraceEnabled())
                 {
-                    logger.trace("created_report_block," + hashCode()
-                            + "," + System.currentTimeMillis()
-                            + "," + reportBlock.getSSRC()
-                            + "," + reportBlock.getNumLost()
-                            + "," + (reportBlock.getFractionLost() / 256D)
-                            + "," + reportBlock.getJitter()
-                            + "," + reportBlock.getXtndSeqNum());
+                    DiagnosticContext diagnosticContext
```

Comment:
i'm just trying to think if there's a way we can avoid having to get the diagnostics context every time.  it's probably not a big deal, but the extra line each time seems kinda annoying.  if we know it's not going to be null (i assume so since we don't do a null check here?), then what about just changing it to `logger.trace(stream.getDiagnosticsContext().makeTimeSeries...` ?

also, if we only log each key from a single place (like you mentioned about not needing the constants) then does the global 'context' really help us much?  what about each class creating a timeseries object for the timeseries they will log, and then just keeping that around? (rather than calling `makeTimeSeriesPoint` each time?).  So somewhere in this file we'd have:
```
TimeSeries createdReportBlockTimeSeries = diagnosticContext.makeTimeSeries("created_report_block")
```
and then here we'd just have:
```
logger.trace(createdReportBlockTimeSeries.addKey(...).addField(...)
```

just some ideas

Reply:
I updated the code to take into account your 1st suggestion, thanks @bbaldino!

As for the 2nd suggestion, I believe your motivation here is to factor out as much functionality as possible, but I think what you're proposing would make the code a bit too verbose, as it would require new class fields to keep a reference to the time series objects.

To give a bit more context here, the functionality that this PR introduces is for debugging purposes, so, the way I imagine it, the API should make adding a time series as frictionless as possible and basically it shouldn't be a whole lot different from adding a debug print message. The API needs to be flexible, terse and promote spatial locality of the changes required to add a time series.

To me it looks like that we've achieved these goals, but please, let me know if you think the `TimeSeries` abstraction is necessary here and I'd be happy to take your suggestion.